site_files/site.NCAR

```diff
@@ -1,22 +1,32 @@
 if (APPLE)
-    set (THIRD_PARTY_DIR     /usr/local/VAPOR-Deps/2019-Aug			           )
-    set (QTDIR               /usr/local/VAPOR-Deps/2019-Aug/Qt/5.13.2/clang_64 )
+    set (THIRD_PARTY_DIR     /usr/local/VAPOR-Deps/2019-Aug		       )
+    set (QTDIR               ${THIRD_PARTY_DIR}/2019-Aug/Qt/5.13.2/clang_64    )
+    set (CMAKE_PREFIX_PATH   ${QTDIR}/lib/cmake                                )
     list (APPEND CMAKE_PREFIX_PATH ${QTDIR}/lib/cmake                          )
-	set (OSPRAYDIR           ${THIRD_PARTY_DIR}/Ospray                         )
 elseif (UNIX AND NOT APPLE)
-    set (THIRD_PARTY_DIR     /usr/local/VAPOR-Deps/2019-Aug			     )
-    set (QTDIR                /usr/local/VAPOR-Deps/2019-Aug/Qt/5.13.2/gcc_64)
-    set (CMAKE_PREFIX_PATH    ${QTDIR}/lib/cmake )
-    set (Qt5Core_DIR          ${QTDIR}/lib/cmake/Qt5Core )
-    set (QT_QMAKE_EXECUTABLE  ${QTDIR}/gcc_64/bin/qmake )
-    set (OSPRAYDIR            ${THIRD_PARTY_DIR}/Ospray )
+    cmake_host_system_information( RESULT hostName QUERY HOSTNAME              )
+    if ( ${hostName} MATCHES "casper" )
+        message( "FOUND CASPER" )
+    	set (THIRD_PARTY_DIR /glade/p/cisl/vast/vapor/third-party/2020-Apr      )
+        set (QTDIR /lib )
+        #set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -stdlib=/glade/u/apps/dav/opt/gnu/9.1.0/lib64/libstdc++")
```

Comment:
Do we need this?

Reply:
No, thanks for spotting it. There's not site file change now.

Reply:
I was referring to the commented out CXX flags but that’s fine